src/code42cli/cmds/alerts.py

```diff
@@ -14,20 +12,22 @@
 import code42cli.options as opt
 from code42cli.cmds.search.cursor_store import AlertCursorStore
 from code42cli.cmds.search.extraction import handle_no_events
+from code42cli.cmds.search.options import server_options
 from code42cli.logger import get_logger_for_server
 from code42cli.options import format_option
-from code42cli.options import server_options
 from code42cli.output_formats import JsonOutputFormat
 from code42cli.output_formats import OutputFormatter
 
 
-SEARCH_DEFAULT_HEADER = OrderedDict()
-SEARCH_DEFAULT_HEADER["name"] = "RuleName"
-SEARCH_DEFAULT_HEADER["actor"] = "Username"
-SEARCH_DEFAULT_HEADER["createdAt"] = "ObservedDate"
-SEARCH_DEFAULT_HEADER["state"] = "Status"
-SEARCH_DEFAULT_HEADER["severity"] = "Severity"
-SEARCH_DEFAULT_HEADER["description"] = "Description"
+def _get_search_default_header():
```

Comment:
Don't we need this `dict` to be an `OrderedDict`?

Reply:
Similarly in other places?

Reply:
No, since we dropped support for py3.5, there is no point in using `OrderedDict` anymore